alacritty.yml

```diff
@@ -480,62 +481,118 @@
 # - `action`: Execute a predefined action
 #
 #   - ToggleViMode
+#       Enable or Disable Using Vi key bindings
 #   - SearchForward
+#       Start a forward buffer search
 #   - SearchBackward
+#       Start a backward buffer search
 #   - Copy
+#       Copy the selection to the clipboard
 #   - Paste
+#       Paste from the system clipboard
 #   - PasteSelection
+#       Paste from the selection buffer
 #   - IncreaseFontSize
+#       Increase font size
 #   - DecreaseFontSize
+#       Decrease font size
 #   - ResetFontSize
+#       Reset font size to config value
 #   - ScrollPageUp
+#       Scroll Exactly One Page Up
 #   - ScrollPageDown
+#       Scroll Exactly One Page Down
 #   - ScrollHalfPageUp
+#       Scroll a half page up
 #   - ScrollHalfPageDown
+#       Scroll a half page down
 #   - ScrollLineUp
+#       Scroll one line up
 #   - ScrollLineDown
+#       Scroll one line down
 #   - ScrollToTop
+#       Scroll all the way to the top
 #   - ScrollToBottom
+#       Scroll all the way to the bottom
 #   - ClearHistory
+#       Clear the display buffer(s) to remove history
 #   - Hide
+#       Hide the Alacritty window
 #   - Minimize
+#       Minimize the Alacritty window
 #   - Quit
+#       Quit Alacritty
 #   - ToggleFullscreen
+#       Toggle full screen
 #   - SpawnNewInstance
+#       Spawn a new instance of Alacritty
 #   - ClearLogNotice
+#       Clear warning and error notices
 #   - ClearSelection
+#       Clear active selection
 #   - ReceiveChar
+#       Allow receiving character input
 #   - None
+#       No action
 #
 #   (`mode: Vi` only):
 #   - Open
+#       Attempt to open the element under the cursor, like a URL. See also url.launcher.
 #   - Up
+#       Move one character up
 #   - Down
+#       Move one character down
 #   - Left
+#       Move one character left
 #   - Right
+#       Move one character right
 #   - First
+#       Move to the start of line
 #   - Last
+#       Move to the end of line
 #   - FirstOccupied
+#       Move to the first non-empty cell
 #   - High
+#       Move to the top of the screen
 #   - Middle
+#       Move to the middle of the screen
 #   - Low
+#       Move to the bottom of the screen
 #   - SemanticLeft
+#       Move to start of semantically separated word
 #   - SemanticRight
+#       Move to start of next semantically separated word.
```

Comment:
You started adding punctuation here. I'd assume that it was by mistake?

Generally I think it would be in line with Alacritty's other internal and external documentation to add periods to all of these.

Reply:
In the last draft, I tried to consistently add punctuation.